examples/all-clusters-app/esp32/README.md

```diff
@@ -160,73 +174,60 @@ your network configuration. To erase it, simply run.
 
           $ idf.py -p /dev/tty.SLAB_USBtoUART erase_flash
 
-### Setting up Python Controller
-
-Once ESP32 is up and running, we need to set up a device controller to perform
-commissioning and cluster control.
-
--   Set up python controller.
+#### Commission a device using chip-tool
 
-           $ cd {path-to-connectedhomeip}
-           $ ./scripts/build_python.sh -m platform
+To initiate a client commissioning request to a device, run the built executable
+and choose the pairing mode.
 
--   Execute the controller.
+#### Commission a device over BLE
 
-           $ source ./out/python_env/bin/activate
-           $ chip-device-ctrl
+Run the built executable and pass it the discriminator and pairing code of the
+remote device, as well as the network credentials to use.
 
-### Commissioning over BLE
+The command below uses the default values hard-coded into the debug versions of
+the ESP32 all-clusters-app to commission it onto a Wi-Fi network:
 
--   Establish the secure session over BLE. BLE is the default mode in the
-    application and is configurable through menuconfig.
+    $ chip-tool pairing ble-wifi ${NODE_ID_TO_ASSIGN} ${SSID} ${PASSWORD} 0 20202021 3840
 
-         - chip-device-ctrl > ble-scan
-         - chip-device-ctrl > connect -ble 3840 20202021 135246
+or equivalently:
 
-         Parameters:
-         1. Discriminator: 3840 (configurable through menuconfig)
-         2. Setup-pin-code: 20202021 (configurable through menuconfig)
-         3. Node ID: Optional.
-            If not passed in this command, then it is auto-generated by the controller and displayed in the output of connect.
-            The same value should be used in the next commands.
-            We have chosen a random node ID which is 135246.
+    $ chip-tool pairing ble-wifi 17 hex:787980 hex:736563726574 0 20202021 3840
```

Comment:
What is this? If not required, please remove

Reply:
done